ui/raidboss/data/05-shb/dungeon/dohn_mheg.txt

```diff
@@ -76,7 +76,7 @@ hideall "--sync--"
 
 ### Aenc Thon, Lord of the Lengthsome Gate
 # -p 35A4:1011 34D2:1300 -ii 34EC 3681 3396
-1000.0 "--sync--" sync /00:0839:The throne room will be sealed off/ window 1000,0
+1000.0 "--sync--" sync /00:0839:The Throne Room will be sealed off/ window 1000,0
```

Comment:
Is this correct? I think this is lower case?

Reply:
![Cap 2020-02-06 19-24-44-700](https://user-images.githubusercontent.com/37621276/73928395-5c9b2100-4916-11ea-8047-fafc9a0a7c45.png)
I just followed the csv file 'PlaceName'. I think it is odd that all others are capital but only it is lower case. If lower case is correct, I'll fix.

Reply:
aren't the sync caseinsensitive anyways?
As long as the timelineReplacments are the same case it shouldn't be a problem right?

Reply:
Yes, the sync is case insensitive, but it also seems odd to me to change the sync from something that had the correct case to something that does not.  In game for English, it is definitely `The throne room will be sealed off`.

Reply:
that really odd.
sorry anyway.